_pages/integrations/arize.md

```diff
@@ -0,0 +1,253 @@
+
+In this guide, we'll show you how to integrate Arize with Algorithmia so you can bring their real-time monitoring capabilities to your algorithms. To make use of this integration, you'll need to have an [Arize account](https://arize.com/request-access/) configured.
+
+The following code is intended to be executed in a Jupyter notebook or on a training platform external to Algorithmia. {:.notice-info}
+
+## Training and saving your model
+To demonstrate the end-to-end workflow, you'll first walk through training a simple scikit-learn model, and then you'll see how to deploy that model on Algorithia and send metrics to Arize from within your algorithm.
+
+As with any Algorithmia algorithm, you can use the platform and tools of your choice for training your model. The code below representes one possible training workflow in a Jupyter notebook; [navigate to the notebook](https://colab.research.google.com/drive/1ikuPSgHXe-LUAYBp2m696XCRJ6ZpYalj?usp=sharing#scrollTo=tDLpUDEXgta2) to work with this code and to see example output.
+
+In your training environment, you'll first need to install the following third-party libraries using pip or the tool of your choice:
+
+{% highlight python %}
+algorithmia>=1.0.0,<2.0
+joblib==1.0.1
+numpy>=1.0
+pandas<2.0,>=0.25.3
+scikit-learn<1.0
+shap==0.39.0
+{% endhighlight %}
+
+Train your model, generate some predictions, and then serialize the trained model:
+
+{% highlight python %}
+import os
+
+import Algorithmia
+import joblib
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+
+# Load and split data.
+from sklearn import datasets
+data = datasets.load_breast_cancer()
+X, y = datasets.load_breast_cancer(return_X_y=True)
+X, y = (
+    pd.DataFrame(X.astype(np.float32), columns=data["feature_names"]),
+    pd.Series(y)
+)
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+
+# Fit a simple logistic regression model.
+clf = LogisticRegression(max_iter=3000, verbose=False).fit(X_train, y_train)
+
+# Generate predictions from the model.
+y_train_pred = clf.predict(X_train)
+y_test_pred = clf.predict(X_test)
+
+# Create a directory to save serialized model files.
+model_dir = "models/"
+if not os.path.exists(model_dir):
+    os.mkdir(model_dir)
+
+# Serialize the model.
+model_file = "Algorithmia_Tutorial_Model.joblib"
+model_file_path = model_dir+model_file
+joblib.dump(clf, model_file_path)
+{% endhighlight %}
+
+## Uploading your trained model to Algorithmia
+Use the following code to upload your model to a hosted data collection on Algorithmia, without ever leaving your training envronemnt. Note that if you're running [Algorithmia Enterprise](/enterprise), you'll need to specify the API endpoint `CLUSTER_DOMAIN` when you create the Algorithmia `client` object; if not, delete the references to this variable.
```

Comment:
```suggestion
Use the following code to upload your model to a hosted data collection on Algorithmia, without ever leaving your training environment. Note that if you're running [Algorithmia Enterprise](/enterprise), you'll need to specify the API endpoint `CLUSTER_DOMAIN` when you create the Algorithmia `client` object; if not, delete the references to this variable.
```

Reply:
lol wow really blew it on the spelling there